packages/tooltips/src/elements/Tooltip.tsx

```diff
@@ -25,30 +25,37 @@ import { StyledTooltipWrapper, StyledTooltip, TOOLTIP_SIZE, TOOLTIP_TYPE } from
 export interface ITooltipProps
   extends Partial<ThemeProps<DefaultTheme>>,
     React.HTMLAttributes<HTMLDivElement> {
-  /** Appends the tooltip to the body element */
+  /** Appends the tooltip to the element provided */
   appendToNode?: Element;
+  /** Adds an arrow to the tooltip */
   hasArrow?: boolean;
-  /** Milliseconds of delay before open/close of tooltip is initiated */
+  /** Adds milliseconds of delay to the opening and closing of the tooltip */
   delayMS?: number;
-  /** Whether Popper should update based on DOM resize events */
+  /** Allows the tooltip to reposition during browser resize events */
   eventsEnabled?: boolean;
+  /** Sets the ID of the tooltip */
   id?: string;
+  /** Defines the content of the tooltip */
   content: React.ReactNode;
   /**
-   * These placements differ from the default naming of Popper placements to help
+   * Adjusts the placement of the tooltip. Placement values differ from Popper placement values to
    * assist with RTL layouts.
    **/
   placement?: GARDEN_PLACEMENT;
-  /** See Popper [documentation](https://popper.js.org/docs/v1/#modifiers--codeobjectcode) for details */
+  /** Passes configurations to the [Popper instance](https://popper.js.org/docs/v1/#modifiers--codeobjectcode) */
   popperModifiers?: Modifiers;
+  /** Adjusts the padding and font size */
   size?: TOOLTIP_SIZE;
+  /** Applies either light or dark styling */
```

Comment:
This feels a wee bit redundant based on what we see here: https://garden.zendesk.com/components/tooltip#tooltip

Reply:
Would something like "Specifies the tooltip type" be any better?

Reply:
Yeah, I think so.